File: SDKs/sportsdata.io/nhl-v3-stats/1.0/go/pkg/models/operations/playersbyteam.go
package operations




type PlayersByTeamFormatEnum string

const (
    PlayersByTeamFormatEnumXML PlayersByTeamFormatEnum = "XML"
PlayersByTeamFormatEnumJSON PlayersByTeamFormatEnum = "JSON"
)


type PlayersByTeamPathParams struct {
    Format PlayersByTeamFormatEnum `pathParam:"style=simple,explode=false,name=format"`
    Team string `pathParam:"style=simple,explode=false,name=team"`
    
}

type PlayersByTeamRequest struct {
    PathParams PlayersByTeamPathParams 
    
}

type PlayersByTeamResponse struct {
    ContentType string 
    Players []interface{} 
    StatusCode int64 
    
}

File: SDKs/sportsdata.io/nhl-v3-stats/1.0/typescript/src/sdk/models/operations/playersbyteam.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";


export enum PlayersByTeamFormatEnum {
    Xml = "XML",
    Json = "JSON"
}


export class PlayersByTeamPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=format" })
  format: PlayersByTeamFormatEnum;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=team" })
  team: string;
}


export class PlayersByTeamRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: PlayersByTeamPathParams;
}


export class PlayersByTeamResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  players?: any[];

  @SpeakeasyMetadata()
  statusCode: number;
}
